pkg/google/gcp.go

```diff
@@ -174,13 +175,6 @@ func (g *GCP) Describe(ch chan<- *prometheus.Desc) {
 	ch <- collectorLastScrapeErrorDesc
 	ch <- collectorDurationDesc
 	ch <- collectorLastScrapeTime
-	for _, c := range g.collectors {
-		if err := c.Describe(ch); err != nil {
-			g.logger.LogAttrs(context.Background(), slog.LevelError, "Error calling describe",
-				slog.String("message", err.Error()),
-			)
-		}
-	}
```

Comment:
We `c.Describe` upfront for AWS & Azure as well, but it's been removed for GCP. Was this intentional cleanup? Should we revert this change?